sparse/impl/KokkosSparse_spgemm_imp_outer.hpp

```diff
@@ -570,7 +570,7 @@ void KokkosSPGEMM<HandleType, a_row_view_t_, a_lno_nnz_view_t_,
       Kokkos::view_alloc(Kokkos::WithoutInitializing, "transpose_col_values"),
       entriesA.extent(0));
 
-  KokkosKernels::Impl::transpose_matrix<
+  KokkosSparse::Impl::transpose_matrix<
```

Comment:
@lucbv: How did this work before?

Reply:
What do you mean, it should have been changed when I moved the transpose operation from the Kernels namespace to the Sparse namespace.
If that is not the case it probably points at code that compiles based on configure option that are not exercised in our builds?